ui/package.json

```diff
@@ -21,6 +21,7 @@
     "@antv/g6": "^4.8.21",
     "@antv/g6-react-node": "^1.4.5",
     "@oceanbase/ui": "^0.2.13",
+    "@openapitools/openapi-generator-cli": "^2.13.5",
```

Comment:
This should be in devDependencies.

Reply:
Run automatic generation, no changes are recommended